pages/apis/rest_api/pipelines.md.erb

```diff
@@ -330,21 +328,9 @@ Required [request body properties](/docs/api#request-body-properties):
     <td>The repository URL.<p class="Docs__api-param-eg"><em>Example:</em> <code>"git@github.com:acme-inc/my-pipeline.git"</code></p></td>
   </tr>
   <tr>
-    <th><code>steps</code></th>
+    <th><code>configuration</code></th>
     <td>
-      An array of the build pipeline steps.
-      <p class="Docs__api-param-eg">
-        <em><a href="/docs/pipelines/command-step">Command</a>:</em> <code>{ "type": "script", "name": "Script", "command": "command.sh" }</code>
-      </p>
-      <p class="Docs__api-param-eg">
-        <em><a href="/docs/pipelines/wait-step">Wait</a> for all previous steps to finish:</em> <code>{ "type": "waiter" }</code>
-      </p>
-      <p class="Docs__api-param-eg">
-        <em><a href="/docs/pipelines/block-step">Block</a> pipeline (see the <a href="/docs/apis/rest-api/jobs#unblock-a-job">job unblock API</a>):</em> <code>{ "type": "manual" }</code>
-      </p>
-      <p class="Docs__api-param-eg">
-        <em><a href="/docs/pipelines/trigger-step">Trigger</a> pipeline:</em> <code>{ "type": "trigger", "trigger_project_slug": "a-pipeline" }</code>
-      </p>
+      The YAML pipeline that consists of the build pipeline steps.<p class="Docs__api-param-eg"><em>Example:</em> <code>"steps:\n - command: \"script/release.sh\"\n "Build \:package\:"\n""</code>
```

Comment:
This fails for me with

```
{
  "message": "Problems parsing JSON"
}
```

Reply:
-d '{
    "name": "My Pipeline",
    "repository": "git@github.com:acme-inc/my-pipeline.git",
    "configuration": "steps:\n - command: \"script/release.sh\"\n " 
  }'

Does work

Reply:
Same as above, let's have a catch-up and chat about it

Reply:
Good pick up, fixed